index.html

```diff
@@ -1,26 +1,27 @@
 
-Course Resources Useful tools
-
-<!-- First link -->
-https://loremipsumgenerator.com/generator/?n=1&t=p Lorem ipsum generator
-
-<!-- Second link  -->
-https://micro-editor.github.io/ Micro: Terminal Text Editor HTML
-
-<!-- First Link -->
-http://www.google.com/search?q=html%20is%20awesome Look for HTML
-
-<!-- Second link -->
-https://developer.mozilla.org/en-US/docs/Web/HTML MDN HTML portal
-
-<!-- Third Link -->
-https://developer.mozilla.org/en-US/docs/Web/HTML/Element MDN HTML element reference
-
-<!-- Fourth Link -->
-https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references HTML Entity reference table
-
-<!-- Fifth Link -->
-https://validator.w3.org/#validate_by_upload+with_options HTML validator
-
-<!-- Sixth Link -->
-https://developer.mozilla.org/en-US/docs/Learn MDN HTML Tutorial
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Course resources</h1></h1>
+
+
```

Comment:
In general: Please check your indention / Format. Using Prettier will do that for you "on save".